index.js

```diff
@@ -190,26 +180,56 @@ exports.eejsBlock_styles = function (hook_name, args, cb) {
 };
 
 function validateSettings() {
```

Comment:
Note that returning `false` with this function gives a gritter error telling users to alert the admin that they should see an error in their logs. It doesn't tell the users what the error is.